include/dxc/DxilContainer/DxilPipelineStateValidation.h

```diff
@@ -266,7 +274,8 @@ struct PSVComponentMask {
     if (ComponentIndex < NumVectors * 4)
       Mask[ComponentIndex >> 5] &= ~(1 << (ComponentIndex & 0x1F));
   }
-  bool IsValid() { return Mask != nullptr; }
+  bool IsValid() const { return Mask != nullptr; }
+  void print(llvm::raw_ostream &, const char *, const char *) const;
```

Comment:
Looks like there are two different naming conventions being used here.  It'd be good to try and figure out what one if most prevalent in this file and follow that one?  Based on a quick look, it seems like that would make this `Print`.

This applies to all the new methods in this PR, I won't call them all out, although if we agree we should follow this convention and you let me know you think you've updated them all I could do that if you like.

Reply:
Updated.